lib/galaxy/tools/parameters/basic.py

```diff
@@ -151,7 +151,7 @@ def value_to_display_text( self, value, app ):
         Convert a value to a text representation suitable for displaying to
         the user
         """
-        if value:
+        if value is not None:
             return unicodify( value )
         return "Not available."
```

Comment:
But not being available means something entirely different than a purposely empty string. A purposely empty string is completely available, as an empty string value.